celer/tests/test_mtl.py

```diff
@@ -199,5 +202,52 @@ def test_GroupLassoCV(sparse_X):
     check_estimator(clf)
 
 
+def test_weights_group_lasso():
+    # generate data
+    sparse_X = 1
```

Comment:
pass directly as parameter and use True. Better yet, parametrize the test with pytest.mark.parametrize to use both sparse_X=Frue and False